src/coredump/stacktrace.c

```diff
@@ -31,6 +31,29 @@ struct stack_context {
         Set **modules;
 };
 
+static struct stack_context *stack_context_destroy(struct stack_context *c) {
```

Comment:
I think it's more reasonable to attach the pointer to the type, i.e. `static struct stack_context* stack_context_destroy`. The function returns a pointer, so `struct stack_context*` is the thing being returned.

(It would look even better if `struct stack_context` was renamed to `StackContext` using our usual style.)